src/homepageExperience/components/steps/arduino/PrepareIde.tsx

```diff
@@ -0,0 +1,44 @@
+// Libraries
+import React from 'react'
+
+export const PrepareIde = () => {
+  return (
+    <>
+      <h1>Prepare Arduino IDE</h1>
+      <p>
+        The easiest way to get started with the InfluxDB Arduino client is with
+        the ESP8266 or ESP32 board.
+      </p>
+      <p>
+        If you haven't already, add the board you wish to use (ESP8266 or ESP32)
+        to the Arduino IDE by following these steps:
+      </p>
+      <p>For ESP8266:</p>
+      <ol style={{fontSize: '16px'}}>
```

Comment:
Since we have this style in two places, its a good idea to either make a variable for it, or define it in the style sheet 👍

Reply:
Good point, I'll add the fontweight to this style too